hendrix/test/performance/pyramid_with_crosstown_traffic.py

```diff
@@ -49,9 +49,10 @@ def cpu_heavy(heft, label=None):
             time.sleep(0)
 
         if count == end:
-            print("%s done: %s" % (label, time.time() - timer_start))
+            print "%s done: %s" % (label, time.time() - timer_start))
```

Comment:
Yeah, I'm not sure what @JamesTFarrington was doing here; unfortunately this is not yet run by the regular test suite.